runtime/handler_test.go

```diff
@@ -178,36 +178,68 @@ func (c *CustomMarshaler) NewDecoder(r io.Reader) runtime.Decoder     { return c
 func (c *CustomMarshaler) NewEncoder(w io.Writer) runtime.Encoder     { return c.m.NewEncoder(w) }
 func (c *CustomMarshaler) ContentType(v interface{}) string           { return "Custom-Content-Type" }
 
+// MarshalerStreamContentType implements Marshaler, but with the addition of a custom StreamContentType.
+type MarshalerStreamContentType struct {
```

Comment:
Can we make this unexported?

Reply:
Done. Was following the pattern of `CommonMarshaler` above, but I'm not attached to it.